archive/2022-07-RStudio-conf/_metadata.yml

```diff
@@ -0,0 +1,2 @@
+execute:
+  freeze: true 
```

Comment:
This automatically freezes the results so rendering new materials does not affect things. 

In the Jobs window, new/updated content in `slides` will be processed as needed and it will skip over the ones in this archive directory: 

```
<snip> processed files <snip>

output file: annotations.knit.md

[10/18] archive/2022-07-RStudio-conf/01-introduction.qmd
[11/18] archive/2022-07-RStudio-conf/02-data-budget.qmd
[12/18] archive/2022-07-RStudio-conf/03-what-makes-a-model.qmd
[13/18] archive/2022-07-RStudio-conf/04-evaluating-models.qmd
[14/18] archive/2022-07-RStudio-conf/05-feature-engineering.qmd
[15/18] archive/2022-07-RStudio-conf/06-tuning-hyperparameters.qmd
[16/18] archive/2022-07-RStudio-conf/07-wrapping-up.qmd
[17/18] archive/2022-07-RStudio-conf/annotations.qmd
[18/18] archive/2022-07-RStudio-conf/index.qmd
```